packages/xo-web/src/common/render-xo-item.js

```diff
@@ -317,6 +317,9 @@ const xoItemToRender = {
         {backup.mode}
       </span>{' '}
       <span className='tag tag-warning'>{backup.remote.name}</span>{' '}
+      <span className='tag tag-success'>
+        {backup.jobName !== undefined ? backup.jobName : '?'}
```

Comment:
Don't you think a simple `?` may confuse users?

Reply:
I tried to keep it short but it may indeed be confusing to someone who doesn't know what the green pills are. "unknown job" should be fine.